### PR TITLE
TMDM-14505 MDM connection in metadata randomly failed with error 'Failed to connect to MDM, please change MDM settings'

### PR DESCRIPTION
**Pull request by sbliu**
https://jira.talendforge.org/browse/TMDM-14505.

Add studio token to created mdm metadata connection.

### CODE REVIEW

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/service/IMDMWebServiceHook.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// ============================================================================
+//
+// Copyright (C) 2006-2019 Talend Inc. - www.talend.com
```
Comment (lpw82gf) [inline main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/service/IMDMWebServiceHook.java]: Should be ` 2006-2020` now

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/service/IMDMWebServiceHook.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+//
+// ============================================================================
+package org.talend.core.service;
+
+import java.util.Map;
+
+import org.talend.core.IService;
+
+/**
+ * created by HHB on 2013-12-23 Detailled comment
```
Comment (lpw82gf) [inline main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/service/IMDMWebServiceHook.java]: by `HHB` ?

File: test/plugins/org.talend.metadata.managment.test/src/org/talend/metadata/managment/mdm/S60MdmConnectionHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
+// ============================================================================
+//
+// Copyright (C) 2006-2019 Talend Inc. - www.talend.com
```
Comment (lpw82gf) [inline test/plugins/org.talend.metadata.managment.test/src/org/talend/metadata/managment/mdm/S60MdmConnectionHelperTest.java]: to `2006-2020`

File: main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/service/IMDMWebServiceHook.java (hunk 1 above)
Comment (cmeng-talend) [reply main/plugins/org.talend.core.runtime/src/main/java/org/talend/core/service/IMDMWebServiceHook.java]: seems it is still not updated yet

**Pull request merged**